minari/dataset/minari_dataset.py

```diff
@@ -218,6 +218,24 @@ def sample_episodes(self, n_episodes: int) -> Iterable[EpisodeData]:
         episodes = self._data.get_episodes(indices)
         return list(map(lambda data: EpisodeData(**data), episodes))
 
+    def iterate_episodes(
+        self, episode_indices: Optional[Union[np.ndarray, List[int]]] = None
+    ) -> Iterator[EpisodeData]:
+        """Iterate over episodes from the dataset in ascending order of indexes.
+
+        Args:
+            episode_indices (Optional[Union[np.ndarray, List[int]]], optional): episode indices to iterate over.
+        """
+        if episode_indices is None:
+            episode_indices = self._episode_indices
+
+        if isinstance(episode_indices, np.ndarray) and episode_indices.ndim != 1:
+            raise RuntimeError("The indices must have a ndim of 1.")
+
+        for episode_index in sorted(list(episode_indices)):
```

Comment:
Hey, thanks for the PR!

Any reason to sort the indices?

Reply:
Hi @younik! Just that the order of indices is always deterministic. I thought this would save the user from having to think in what order the indexes of the episodes would be passed to iterate.

Reply:
I would avoid sorting because if I define the argument `episode_indices` in a specific order, I expect the output to match it.

Reply:
I agree with @younik. @Howuhh can you remove the sorting?

Reply:
@rodrigodelazcano Yeah, it would be better without sorting

Reply:
Removed sorted